src/AutoMerge/source.extension.vsixmanifest

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
 	<Metadata>
-		<Identity Id="AutoMerge.VS2017.F05BAC3E-6794-4A9E-9EE7-1B8A200778EE" Version="0.2.6.9" Language="en-US" Publisher="Kulikov Denis" />
-		<DisplayName>Auto Merge for Visual Studio 2017</DisplayName>
+		<Identity Id="AutoMerge.VS2019.F05BAC3E-6794-4A9E-9EE7-1B8A200778EE" Version="0.2.6.10" Language="en-US" Publisher="Kulikov Denis" />
```

Comment:
I believe this file should not be changed, introducing VS-specific manifest "source.vs17.0.extension.vsixmanifest" is enough.

Reply:
This file is automatically changed by the build process - so no need to undo.